src/utilities/Utilities.js

```diff
@@ -156,3 +156,26 @@ export const isInViewport = (elem) => {
       (window.innerWidth || document.documentElement.clientWidth)
   )
 }
+
+export const isTopInViewport = (elem) => {
+  // Adapted from https://gomakethings.com/how-to-test-if-an-element-is-in-the-viewport-with-vanilla-javascript/
+  if (!elem) return false
+  const bounding = elem.getBoundingClientRect()
+  return (
+    bounding.top >= 0 &&
+    bounding.top <=
+      (window.innerHeight || document.documentElement.clientHeight)
+  )
+}
+
+export const stickyScrollIntoView = (selector) => {
```

Comment:
Can you parameterize the selector for the stickyHeader too?  maybe `(targetSelector, headerSelector = "#sticky-resource-header")`

Reply:
Done.